models/user_mail.go

```diff
@@ -343,54 +316,32 @@ type SearchEmailResult struct {
 // SearchEmails takes options i.e. keyword and part of email name to search,
 // it returns results in given range and number of total results.
 func SearchEmails(opts *SearchEmailOptions) ([]*SearchEmailResult, int64, error) {
-	// Unfortunately, UNION support for SQLite in xorm is currently broken, so we must
-	// build the SQL ourselves.
-	where := make([]string, 0, 5)
-	args := make([]interface{}, 0, 5)
-
-	emailsSQL := "(SELECT id as sortid, uid, email, is_activated, 0 as is_primary " +
-		"FROM email_address " +
-		"UNION ALL " +
-		"SELECT id as sortid, id AS uid, email, is_active AS is_activated, 1 as is_primary " +
-		"FROM `user` " +
-		"WHERE type = ?) AS emails"
-	args = append(args, UserTypeIndividual)
-
+	var cond builder.Cond = builder.Eq{"user.`type`": UserTypeIndividual}
 	if len(opts.Keyword) > 0 {
-		// Note: % can be injected in the Keyword parameter, but it won't do any harm.
-		where = append(where, "(lower(`user`.full_name) LIKE ? OR `user`.lower_name LIKE ? OR emails.email LIKE ?)")
 		likeStr := "%" + strings.ToLower(opts.Keyword) + "%"
-		args = append(args, likeStr)
-		args = append(args, likeStr)
-		args = append(args, likeStr)
+		cond = cond.And(builder.Or(
+			builder.Like{"lower(`user`.full_name)", likeStr},
+			builder.Like{"`user`.lower_name", likeStr},
+			builder.Like{"email_address.email", likeStr},
```

Comment:
Should this be lower too?

```suggestion
			builder.Like{"email_address.lower_email", likeStr},
```

Reply:
Done.